minibot/new_minibot.py

```diff
@@ -94,14 +97,16 @@ def parse_command(cmd, tcpInstance):
                     file_dir + "/scripts/" + script_name, 'w+')
                 file.write(program)
                 file.close()
-                spawn_script_process(script_name, tcpInstance)
-            except Exception as exception:
-                print("Exception occurred")
-
-    elif key == "STOP":
-        if current_process is not None:
-            current_process.terminate()
-        stop = True
+                """
```

Comment:
Use comments like _# Run the Python program_ instead of doc string like comments within functions.